### ISSUE TITLE
[Feature Request]: Some notes about running on Mac

### ISSUE THREAD

**Issue by bitwes** [DmitriySalnikov/GodotPCKExplorer#14]
### Feature description

Not a bug or a feature really, but I just got this running on a Mac and thought I'd let you know how it went.

1.  Had to upgrade dotnet (https://dotnet.microsoft.com/en-us/download).  It might be nice to note the version required, unless I missed this.
2. My Mac got really angry about `mbedTLS_AES.dylib` and threw up a dialog about 3 or 4 times when trying to run it.  [Stackoverflow](https://stackoverflow.com/questions/66891065/libprism-sw-dylib-cannot-be-opened-because-the-developer-cannot-be-verified-o) informed me on how to live dangerously.  I made the following bash script to make my life a little easier and every so slightly more secure:
```bash
# -----------
# EDIT:  You probably shouldn't do this as it didn't work the way I expected it to (see later posts)
# -----------
function pck_explorer(){
  # This has to be disabled for the pck explorer to open on mac.  I barely read 
  # the stack overflow link, so I'm just trusting that everything is legit here.
  sudo spctl --master-disable
  dotnet <path to extracted dir>/GodotPckExplorer.Console.dll "$@"
  # turning this back on seemed like a good idea
  sudo spctl --master-enable
}
```
4. I tried using `-o` on the mac and the message told me that the UI only worked on Windows.  Since I really wasn't sure what this tool did, my first thought was that it didn't work at all on Mac.  This is because I'm lazy and didn't read much.  Others might be lazy too.  I think a better message might be:
"The UI is only supported on Windows but the other CLI options work.  Use them instead...you made it this far, we know you got all the cool CLI skills, go for it". 
Or something like that.

I hope this feedback helps.  Seems like a really nice tool you made.  
Thanks

**Comment by DmitriySalnikov** [comment]
> 1\. It might be nice to note the version required, unless I missed this.

[Here](https://github.com/DmitriySalnikov/GodotPCKExplorer?tab=readme-ov-file#download) and [here](https://github.com/DmitriySalnikov/GodotPCKExplorer/releases/tag/1.4.0#:~:text=USDT%2DTRC20%20TEw934PrsffHsAn5M63SoHYRuZo984EF6v-,Changes,-The%20main%20change)

> 2\. My Mac got really angry about `mbedTLS_AES.dylib`

I just downloaded the version from the releases page and ran it with no problems on macOS 12.
![image](https://github.com/DmitriySalnikov/GodotPCKExplorer/assets/7782218/a24cde48-3716-4acd-af79-ce447d12b0b8)

Did this protection come out after macOS 12? Or do I need real hardware to get the same error?

I found that you probably need to unlock the file in `System Preferences > Security & Privacy`. This may be due to the fact that the file was not downloaded via Safari. Idk.

> 4\. I tried using `-o` on the mac and the message told me that the UI only worked on Windows.

I thought this was a fairly informative message `The UI is only supported on Windows`.
But I agree that I could do it in a more informative way. I can just not display information about this command in Linux and macOS.

**Comment by bitwes** [comment]
It should be said, that by the time I found this repo I was exceptionally annoyed.  I was trying to track down a bug with `DirAccess` and exported projects, and "I just wanted to look inside a PCK, why isn't this easier to do".  So I didn't read much, just tried to get it working.  This feedback was to help idiot proof the tool, since I was in "just get this working" mode and not RTFM mode.

> Did this protection come out after macOS 12? Or do I need real hardware to get the same error?

I'm not sure when it showed up, but I'm on MacOS Ventura 13.4.1 (22F82).  

> I found that you probably need to unlock the file ....

Yea, I saw that too.  I just tried to get a screen capture of the error, but after removing `sudo spctl --master-disable` from my function, it didn't show up.  As mentioned above, this was all done hastily and I probably should have understood what `sudo spctl --master-disable` actually did.  It was easier to add that to a little function than it was to get lost in stupid macOS settings dialogs...I thought, heh.

The options on the dialog were "Move to trash" and "Cancel".  The "Move to trash" option sounded like it would break things even further, but someone might click it.

> I thought this was a fairly informative message

It is a good message for the the option.  That was just the first option I tried because it was at the top of the list and I wanted to "open" a pck.  Maybe the `-o` option should say "open PCK in GUI"?
